web-app/HeimScripts/heatmap/run.r

```diff
@@ -7,6 +7,8 @@ main <- function(max_rows=50){
   if(ncol(df) > 3){
     variances <- apply(df[,3:ncol(df)],1,var) # Calculating variance per probe
     df["SIGNIFICANCE"] <- variances
+    df["MEAN"] <- apply(df[,3:ncol(df)],1,mean, na.rm = T)
+    df["SD"] <- apply(df[,3:ncol(df)],1,sd, na.rm = T)
     df <- df[with(df, order(-SIGNIFICANCE)), ]
```

Comment:
first time I see the use of 'with' in R, so I cannot judge if this works correctly